.github/workflows/pr.yml

```diff
@@ -19,6 +19,6 @@ jobs:
       with:
         workflow: test.yml
         repo: hashicorp/consul-k8s-workflows
-        ref: main
```

Comment:
Does this need to be swapped back before merging?

Reply:
yep, I marked it to be dropped in the commit message